node/how_to_with_npm/react.md

```diff
@@ -0,0 +1,250 @@
+# How to use React with Deno
+
+[React](https://reactjs.org) is the most widely used JavaScript frontend
+framework. It popularized a declarative approach towards designing user
+interfaces, with a reactive data model. Due to its popularity, it's not
+surprising that it's the most requested framework when it comes to building web
+apps with Deno.
+
+This is a tutorial that walks you through building a simple React app with Deno
+in less than five minutes. The app will display a list of dinosaurs. When you
+click on one, it'll take you to a dinosaur page with more details.
+
+![demo of the app]()
+
+Here's the hosted version of the app. View the source here.
+
+## Create Vite Extra
+
+This tutorial will use [Vite](https://vitejs.dev/) to quickly scaffold a Deno
+and React app. Let's run:
+
+```sh
+deno run -A npm:create-vite-extra
```

Comment:
I think this one is ok to run without a version requirement because then you always get the latest information.